### PR TITLE
player.js

### PR DESCRIPTION
**Pull request by iibrahimwt13**

### CODE REVIEW

File: plugins/MusicPlayer/player.js [hunk 1]
```diff
@@ -312,7 +312,7 @@ exports.dequeue = {
     * Pause command.
     *
     * @param msg Original message.
-    * @param suffix Command suffix.
+    * @param suffix Command s>uffix
```
Comment (F9Alejandro) [inline plugins/MusicPlayer/player.js:315]: Changes to words from english to another language doesn't warrant a pull request to the main repo as majority of the users are english speakers. If you want you can make a translation file in yml or json format and can send it over for use in making this bot multilingual.

File: plugins/MusicPlayer/player.js [hunk 2]
```diff
@@ -11,12 +11,12 @@ let MAXIMUM_SONG_BUFFER_SIZE = (options && options.maxSongBufferSize) || 1024 *
...
-    "play",
-    "skip",
+    "oynat",
+    "geç",
     "queue",
     "dequeue",
-    "pause",
-    "resume"
+    "durdur",
+    "devamet"
```
Comment (F9Alejandro) [inline plugins/MusicPlayer/player.js:19]: Changing from english to another language is not supported in pull requests. Try making a lang file for your language to have support added.

**Review by F9Alejandro — COMMENTED** [review]
Try making a lang file for the translations from english to your native language in a lang.yml or json file for better access for multilingual developments.

**Comment by Einarin** [comment]
Closing this as the patch isn't helpful as is. We should look at making command localization possible in the future, but this isn't the way to do it.

**Pull request closed without merging**